pkg/constant/viper_config.go

```diff
@@ -52,4 +52,6 @@ const (
 	CfgKBReconcileWorkers = "KUBEBLOCKS_RECONCILE_WORKERS"
 	CfgClientQPS          = "CLIENT_QPS"
 	CfgClientBurst        = "CLIENT_BURST"
+
+	CfgRegistries = "registries"
```

Comment:
A global flag is hard to config and maintain for the user, why not using the kb manager config CM?

Reply:
It does uses the manager config CM. This flag is the first level key in the config.